templates/paragraphs/paragraph--callout.html.twig

```diff
@@ -0,0 +1,4 @@
+{% extends "@molecules/callout/callout.twig" %}
+  {% block callout__items %}
```

Comment:
This is valid and works well. But I'm wondering if something is inconsistent with how we pass around data.

In this template we are passing an entire Drupal field to the component library. That is `content.field_callout` is a complex object. One that makes sense in Drupal but has a lot of extra stuff not important to rendering the item.

Instead, should we loop over the field content here in Drupal and old pass the values that we need to the component library? Wiring Drupal field value -> component.

Just thinking aloud.

Reply:
Looping the way you mention gets real messy, real fast. Drupal's array isn't very easy to traverse (at least not in Twig) so we've found it's easiest for nested items, and multi-value fields like this to just print the field in a block, and then let the individual item map to a corresponding template. So that's what `paragraph--callout-item.html.twig` is doing.